scripts/auto-rebase/rebase.sh

```diff
@@ -1161,25 +1223,34 @@ case "$command" in
     to)
         # FIXME: Allow more than 3 arguments until pipelines stop passing LVMS value.
```

Comment:
Is this for ART rebase procedure?

Reply:
Yes, the ART pipeline is still passing the extra value.